### DISCUSSION TITLE
Order clusters by a variable

### DISCUSSION THREAD

**Discussion by acpaulo** [cytoscape/cytoscape.js#3058] [Q&A] [answered]
Hi,
I´m using cytoscape to visualise GPSC and poppunk clusters. Each cluster has nodes that are pneumococcal serotypes. Clusters belong to GPSCs. It happens that cytoscape presents these clusters from the major, more nodes to minor ones, single nodes. This means that, for instance, I can have a cluster belonging to GPSC3 in the first line and a smaller one in the fifth line. I would like to order clusters by GPSC. How can I do this? I didn´t find a way to do this in cytoscape or to edit in other programme.
I would appreciate any help.
Thank you in advance.
Cristina Paulo

**Comment by maxkfranz** [comment, marked as answer]
It sounds like you're using the [Cytoscape desktop app](https://cytoscape.org/index.html).  If so, you should take a look at [the help section on its site](https://cytoscape.org/documentation_users.html).

**Reply by acpaulo** [reply]
Thank you. I already found the solution.
